github-page/_data/navigation.yml

```
@@ -113,6 +113,8 @@ guides:
         url: /guides/advanced/plugins
       - title: 'Change _id Type'
         url: /guides/advanced/changeIDType
+      - title: 'Using ObjectId Type'
```

```suggestion
      - title: 'Using ObjectId'
```

try to keep them in sync with the actual title

---

Good point, I think "Using ObjectId Type" makes more sense so I'll change the title.